ci/build_scripts/build.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash -x
```

Comment:
I added `-x` option to print out all steps so that easier to debug in GitHub Actions because the initial scope of the build script was only to be used in the workflows.
However, now the script became more general, and I found `-x` option kills the beauty of the script. Should we remove `-x`?

```suggestion
#!/bin/bash
```

Reply:
Yes a good suggestion. I disabled the verbose mode ("-x") by default, and only activated it in the ci workflow by calling `bash -x <script>`